decidim-forms/app/cells/decidim/forms/step_navigation/show.erb

```diff
@@ -5,7 +5,10 @@
     <%= link_to(
       icon("caret-left", class: "icon--small", role: "img") + " " + t("decidim.forms.step_navigation.show.back"),
       "#",
-      class: "hollow secondary"
+      class: "hollow secondary",
+      data: {
+        toggle: [previous_step_dom_id, current_step_dom_id].join(" ")
```

Comment:
Wow, I was looking for the JS part! 🤩 ❤️

Reply:
It turned out to be much easier than expected! 😄 Luckily, Foundation's `toggler` component allows for multiple toggles from the same element, this allowed me to implement the steps 😄